feat(plugins-doc) update documentation for centreon-plugin-applications-protocol-[cifs,ftp,sftp] plugins (#3269)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-protocol-sftp.md

import TabItem from '@theme/TabItem';

## Pack assets

### Templates

The Monitoring Connector **Protocol SFTP** brings a host template:

* **App-Protocol-Sftp-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Protocol-Sftp-custom" label="App-Protocol-Sftp-custom">

| Service Alias | Service Template                    | Service Description                      |
|:--------------|:------------------------------------|:-----------------------------------------|
| Connection    | App-Protocol-Sftp-Connection-custom | Check connection on a remote SFTP server |

> The services listed above are created automatically when the **App-Protocol-Sftp-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias | Service Template                     | Service Description                  |
|:--------------|:-------------------------------------|:-------------------------------------|
| Files-Count   | App-Protocol-Sftp-Files-Count-custom | Check number of files in a directory |

> | Files-Date    | App-Protocol-Sftp-Files-Date-custom  | Check modified time of files         |

Reviewer: ```suggestion
| Files-Date    | App-Protocol-Sftp-Files-Date-custom  | Check last modified time of files         |
```

> To monitor your SFTP server, you can use pubkey authentication and/or username authentication.

Reviewer: ```suggestion
To monitor your SFTP server, the plugin can either log in using an SSH key, or use username authentication.
```

| EXTRAOPTIONS       | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). | --verbose              |             |

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
is able to monitor a resource using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_protocol_sftp.pl \
	--plugin=apps::protocols::sftp::plugin \
	--mode=scenario \
	--hostname='10.0.0.1'  \

> 	--ssh-username='' \

Reviewer: ```suggestion
	--ssh-username='my-username' \
```

> 	--ssh-password=''  \

Reviewer: ```suggestion
	--ssh-password='my-password'  \
```
